### PR TITLE
Adding more modules to proxy

### PR DESCRIPTION
**Pull request by angeliquemari**
- add reset css sheet, link to from proxy's index.html file
- import styled components to proxy's index.html file
- add div & import statement for 4 additional modules in proxy's index.html file

Note, in order for the proxy to render all components to the page, this update requires the following changes in other modules:
- updating id of the div that a module is rendering to to (i.e. not 'app')
- adding CORS headers in a module's server (for reference see https://github.com/O2znz/photo-gallery/pull/10)
- hard-coding the URL of the get request on component mount for a module (for reference see https://github.com/O2znz/photo-gallery/pull/10, although I passed the url in as a prop but I probably didn't need to do that, it can be put in the component itself)
- replacing import statements in styled component/react component files with `const styled = window.styled;` or equivalent (for reference see https://github.com/O2znz/photo-gallery/pull/11)
- port numbers for every module are mutually exclusive
- importing react and styled-components in the index.html file of a module (this might be optional)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+  <div id="recommendations"></div>
+  <div id="experiences"></div>
   <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
   <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+  <script src="//unpkg.com/styled-components/dist/styled-components.min.js"></script>
   <script src="http://localhost:3001/:listingid/bundle.js"></script>
+  <script src="http://localhost:3002/bundle.js"></script>
+  <script src="http://localhost:3003/bundle.js"></script>
+  <script src="http://localhost:3004/bundle.js"></script>
+  <script src="http://localhost:3005/:listingid/bundle.js"></script>
```
Comment (angeliquemari) [inline public/index.html]: for some reason I could only get this to work with :listingid (on proxy side and module side), we should probably play around with it a bit more @BrewBeast

File: public/index.html [hunk 2]
```diff
@@ -1,11 +1,21 @@
 <!doctype html>
 <head>
   <title>o2znz</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css" />
 </head>
 <body>
   <div id="photo-gallery"></div>
+  <div id="checkout"></div>
```
Comment (angeliquemari) [inline public/index.html:8]: fyi I had to update the div name in modules to get them to match here (since they can't all be called 'app')

**Review by Sam-I-Am92 — APPROVED** [review]
Looks good to me!

File: public/index.html (hunk 1 above)
Comment (angeliquemari) [reply public/index.html]: looks like this self-resolved somehow!

File: public/index.html (hunk 2 above)
Comment (angeliquemari) [reply public/index.html:8]: these changes have been made on the individual modules now

File: public/index.html [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+  <div id="reviews"></div>
+  <div id="recommendations"></div>
+  <div id="experiences"></div>
   <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
   <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+  <script src="//unpkg.com/styled-components/dist/styled-components.min.js"></script>
   <script src="http://localhost:3001/:listingid/bundle.js"></script>
+  <!-- <script src="http://localhost:3002/bundle.js"></script> -->
```
Comment (angeliquemari) [inline public/index.html:16]: this module is breaking all the others, so leaving commented out for now

**Pull request merged**